internal/service/studies/main.go

```diff
@@ -100,21 +115,78 @@ func (s *Service) validateStudyData(ctx context.Context, username types.Username
 	return nil
 }
 
-func (s *Service) CreateStudy(ctx context.Context, user types.User, studyData openapi.StudyCreateRequest) (*types.Study, error) {
+func (s *Service) CreateStudy(ctx context.Context, user types.User, studyData openapi.StudyCreateRequest) (openapi.StudyCreateResponse, error) {
 	if err := s.validateStudyData(ctx, user.Username, studyData); err != nil {
-		return nil, types.NewErrInvalidObject(err)
-	}
-	additionalStudyAdminUsernames := []types.Username{}
-	for _, username := range studyData.AdditionalStudyAdminUsernames {
-		additionalStudyAdminUsernames = append(additionalStudyAdminUsernames, types.Username(username))
+		isValid := false
+		errorMessage := err.Error()
+		return openapi.StudyCreateResponse{
+			IsValid:      &isValid,
+			ErrorMessage: &errorMessage,
+		}, nil
```

Comment:
This approach follows a similar convention to `updateNHSD` where the error message is returned in the response as a custom attribute. 

I found it difficult to wrap my head around separating the Go errors handled by `setError` and the error message returned to the frontend as I thought they would all be handled as one process.

Reply:
> I found it difficult to wrap my head around separating the Go errors handled by setError and the error message returned to the frontend as I thought they would all be handled as one process.

this is fair. there is a distinction though – you'll never see AWS returning you an error with connection details for their database in..